rechunker/algorithm.py

```diff
@@ -3,7 +3,10 @@
 
 try:
     from math import prod
+
+    numpy_prod = False
```

Comment:
Apparently prod is not in math until python 3.8. Maybe we should just always use numpy here? Math has type annotations.